plugins/cookie_manager/lib/src/cookie_mgr.dart

```diff
@@ -8,6 +8,8 @@ const _kIsWeb = bool.hasEnvironment('dart.library.js_util')
     ? bool.fromEnvironment('dart.library.js_util')
     : identical(0, 0.0);
 
+final _setCookieReg = RegExp('(?<=)(,)(?=[^;]+?=)');
```

Comment:
Can you add more comments to help everyone understand how this regular expression works?

Reply:
Comment generated by chatgpt because I am definitely worse in understanding regex than it.